<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-M471DVV" failures="1" tests="3" name="Test" time="15.738" errors="0" timestamp="2025-03-26T16:03:26 IST">
  <testcase classname="day_46.OrangrHRM" name="testLogo" time="0.103"/>
  <testcase classname="day_46.OrangrHRM" name="testUrl" time="0.034">
    <failure type="java.lang.AssertionError" message="expected [https://opensource-demo.orangehrmlive.com] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]">
      <![CDATA[java.lang.AssertionError: expected [https://opensource-demo.orangehrmlive.com] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]
at day_46.OrangrHRM.testUrl(OrangrHRM.java:40)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- testUrl -->
  <testcase classname="day_46.OrangrHRM" name="testTitle" time="0.0">
    <skipped/>
  </testcase> <!-- testTitle -->
</testsuite> <!-- Test -->
